src/main.py

```diff
@@ -40,6 +43,16 @@ def post_slack_message(hook_url, message):
     # print('Response: {}, message: {}'.format(response.status, response.read().decode()))
     return response.status
 
+def publish_sns(sns_topic, event):
+    try:
+        return sns.publish(
+            TargetArn=sns_topic,
+            Message=json.dumps({'default':  json.dumps(event)}),
```

Comment:
zwei mal json dumps? Dann hab man einen scaped string in der message.

Reply:
Das wrappen in default war unnötig. Das wird nur benötigt wenn man pro Target Typ ein anderes Format senden will (sms, email, default...). Die Message an sich muss einmal encoded werden. Habs gefixt